src/components/Sidebar/Sidebar.jsx

```diff
@@ -12,13 +12,24 @@ import { SidebarFooter } from "./Sidebar.Footer";
 import { SidebarHeading } from "./Sidebar.Heading";
 import { SidebarLink, SidebarSeparator } from "./Sidebar.Link";
 import { SidebarMenu } from "./Sidebar.Menu";
+import sidebarScroll from "./SidebarScroll.module.css";
 
 const LeftDrawerCountStyle = {
     // From Figma
     background: "linear-gradient(138.65deg, #583DFF 19.59%, #F849C7 62.96%, #FFC03D 97.07%)",
 };
 
-export const Sidebar = ({ logo, children, className, footer, notifications, isFixed = true, onLogoClick }) => {
+export const Sidebar = ({
+    logo,
+    children,
+    className,
+    footer,
+    notifications,
+    isFixed = true,
+    onLogoClick,
+    isStickyHeader = true,
```

Comment:
not sure, did we need to have Sticky header by default  ?

Reply:
In The ticket mentioned - _the flexible part of the left menu should only be below the settings tab and above the logo_
so everything above the logo is sticky